lib/gears/math/bezier.lua

```diff
@@ -19,6 +19,7 @@ local bezier = {}
 -- @tparam number t The value of the t parameter to evaluate the curve at.
 -- @treturn number The value of `B(t)`.
 -- @staticfct gears.math.bezier.curve_evaluate_at
+-- @see wibox.widget.graph.step_hook
```

Comment:
These make no sense IMO. They're pure math functions with no direct connection to graphs or drawing in general. Many of them aren't even used in the `step_hook` example and probably wouldn't even be useful in such a callback, but were simply added for completeness, because that's what any decent generic bezier curve math library has to have. "@<!-- -->see wikipedia" would make more sense at this point.